### DISCUSSION TITLE
Can't parse the columns from database

### DISCUSSION THREAD

**Discussion by caiqueacct** [feathersjs/feathers#3304] [General]
Hi

The database return a column with JSON data but feathers is returning to the application as string. 

I'm not understanding how to map the columns with the correct datatype.

Any suggestions?

**Comment by 8BallBomBom** [comment]
All of the output data will usually be in a wrapped JSON format.
So if you are storing JSON within the database? then that data will be outputted as a string within the final JSON output.
Converting the string to JSON should do the trick, have a look at [this](https://www.w3schools.com/js/js_json_parse.asp).
